### ISSUE TITLE
Unable to start a scenario

### ISSUE THREAD

**Issue by cielo** [factoriotools/factorio-docker#335]
Hello, 

I have been playing around with running factorio server in docker, and encountered this issue today.

Running factorio with default docker-entrypoint works flawlessly.

However, if I want to start a server from scratch with scenario using `scenario.sh`, I get the following error.

```
0.000 Program arguments: "/opt/factorio/bin/x64/factorio" "--port" "34197" "--start-server-load-scenario" "simple" "--server-settings" "/factorio/config/server-settings.json" "--server-banlist" "/factorio/config/server-banlist.js
on" "--server-whitelist" "/factorio/config/server-whitelist.json" "--use-server-whitelist" "--server-adminlist" "/factorio/config/server-adminlist.json" "--rcon-port" "27015" "--rcon-password" <private> "--server-id" "/factorio/co
nfig/server-id.json" 

+ SERVER_SCENARIO=simple
+ mkdir -p /factorio/saves
+ mkdir -p /factorio/config
+ mkdir -p /factorio/mods
+ mkdir -p /factorio/scenarios

...

   0.347 Loading mod base 0.17.79 (data-updates.lua)
   0.505 Checksum for core: 2035938594
   0.505 Checksum of base: 4156627525
   0.728 Prototype list checksum: 2767119397
   0.794 Info PlayerData.cpp:68: Local player-data.json available, timestamp 1595911666
   0.794 Info PlayerData.cpp:75: Cloud player-data.json unavailable
   0.798 Factorio initialised
   0.799 Error Main.cpp:796: Scenario /opt/factorio/scenarios/simple not found
```

What I noticed is that most of the custom server settings (map-gen/server-settings/etc) are referenced from `/factory/` directory provided as command line arguments shown above.

However, factorio binary seems to looking up the default folder for the scenario, which is located in `/opt/factorio/scenarios` and it cannot find scenario folder as shown in the docker logs above.

I have copied my scenario files under 'scenarios' folder in host machine that is mounted to the docker container, and it should be available in the docker container in `/factorio/scenarios`, but not in `/opt/factorio/scenarios/`.

Is this a known issue? or am I missing a step?

**Comment by SuperSandro2000** [comment]
Can you post the docker run command?

**Comment by cielo** [comment]
I am using a slightly modified version of the command listed in the README.
```
docker run -d \
  -p 34197:34197/udp \
  -p 27015:27015/tcp \
  -v /home/factorio:/factorio \
  --name factorio \
  --restart=always  \
  --entrypoint "/scenario.sh" \
  factoriotools/factorio \
  simple
```

In host machine, config/save/scenario folders are located under /home/factorio.

**Comment by SuperSandro2000** [comment]
Can you test #336 and see if it works now?

**Comment by cielo** [comment]
That did the trick. Thank you for the quick fix!

**Comment by SuperSandro2000** [comment]
Nice to hear that!

**Comment by cielo** [comment]
Hello, I see that 0.18 docker image has been updated recently, but not 0.17 so far yet. Do you happen to know if this change can be published to the 0.17 docker image on docker hub as well?

**Comment by SuperSandro2000** [comment]
I forgot to create a new release tag. Thanks for the reminder. The new version is on its way.
